assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -102,7 +103,11 @@ const CartLineItemRow = ( { lineItem } ) => {
 	const firstImage = images.length ? images[ 0 ] : {};
 
 	return (
-		<tr className="wc-block-cart-items__row">
+		<tr
+			className={ classnames( 'wc-block-cart-items__row', {
```

Comment:
I would have gone with `LoadingMask` but this is a table row and we can’t use divs.